src/App.tsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 
 import './App.scss';
+import { TabsList } from './components/TabsList';
 
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+class App extends React.Component<{}, State> {
+  state = {
+    tabsCopy: [...tabs],
+    selectedTab: 'tab-1',
```

Comment:
if it's id, name it selectedTabId, and better here to store not id, but all tab. So you will be able not to do 'find' method many times